### DISCUSSION TITLE
Buttons not working, display half lit after boot completes

### DISCUSSION THREAD

**Discussion by acidkewpie** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#153] [Q&A] [answered]
Hi, I soldered up a version of this a few months back, and today finally found the guts and motivation to install it. The controller itself seems to work just great, however the physical buttons and display are not working. It shows the temperature just fine during boot, however as soon as the jingle plays, it all goes south, with only a few led's half list and nothing responds.

Please don't laugh at my non-pcb solution too much, and I appreciate it's naturally hard to see what's going on. It does feel like it's something pretty simple though, given it initially IS displaying the temperature. Resistor fun?

![image](https://user-images.githubusercontent.com/3334068/143601590-12ecdb37-6a9c-44ac-97d6-2a68773b4bfb.png)
![image](https://user-images.githubusercontent.com/3334068/143601655-0c65a4f0-9fbb-4889-ab84-f88ba8f388a2.png)
![image](https://user-images.githubusercontent.com/3334068/143601673-4579e5a9-de0b-45f0-8124-a1b5b1f754f0.png)
![image](https://user-images.githubusercontent.com/3334068/143601686-16613db1-282c-46ea-90e5-50aa6ea625a5.png)
![image](https://user-images.githubusercontent.com/3334068/143601712-d8d93bf5-5cf6-4b06-9bc1-7bc7fe49d933.png)

**Comment by visualapproach** [comment, marked as answer]
Compact device! There is a couple of models to choose from in the code. You have to define the right model. If that's not the problem I'd ditch the resistors and replace with wires. Don't pay attention to what the display says before the hello message is showing, it may be data left in the display's memory from earlier.

**Comment by visualapproach** [comment]
And btw, make sure you've entered the wifi credentials in the autoportal.

**Comment by acidkewpie** [comment]
Awesome yes, removing the resistors on the display side seems to have sorted it. Fun surprise when it started up! Reminded me of this... https://www.youtube.com/watch?v=TrryML0XVuY :)

Now to work out the best way to send commands to it on a persistent daily schedule and when my solar battery has power to spare.

**Reply by visualapproach** [reply]
ROTFL 🤣
